### ISSUE TITLE
algolia/webmock does not correctly register request stubs

### ISSUE THREAD

**Issue by hannesfostie** [algolia/algoliasearch-client-ruby#223]
Hey folks,

I've mentioned this to @redox in a PR on one of our apps (or rather, a demo app) and have recreated the issue I've been experiencing in [this PR](https://github.com/hannesfostie/big_articles/pull/2)

The PR shows 2 commits, one being the initial setup for `webmock` and `webmock/algolia` where tests will fail. The second copies all request stubs from the file we required and adds them in the test file itself, making the tests pass.

TL;DR: just using `require 'webmock/algolia'` doesn't actually appear to register any stubs, and I'm not entirely sure why.

**Comment by hannesfostie** [comment]
I just realized this may be a duplicate of #100
